### ISSUE TITLE
1.20.5/.6 and 1.21 plz

### ISSUE THREAD

**Issue by nekOStorm** [Octol1ttle/DamageVignette-Fabric#17]
This mod is useful in many cases, but when I want to install it into Minecraft 1.21, I cant find it

**Comment by leeineian** [comment]
^ (For the current latest version 1.21.4 as well thank you)

**Comment by Keaton42** [comment]
Yes!! I need this so badly!! I use 1.21.1 but 1.21 would also work fine!

**Comment by patinopsu** [comment]
This project is no longer in active development, Sorry
